moped-editor/src/views/projects/projectView/ProjectComponents/ComponentForm.js

```diff
@@ -56,7 +56,7 @@ const validationSchema = yup.object().shape({
   subphase: yup.object().nullable().optional(),
   tags: yup.array().optional(),
   completionDate: yup.date().nullable().optional(),
-  description: yup.string(),
+  description: yup.string().nullable().optional(),
```

Comment:
ah, interesting! it was a validation issue that we weren't bubbling up to the UI. thanks for figuring this out!

Reply:
thanks! yeah it turned out to be a really simple fix :)